### ISSUE TITLE
Failed to activate the line-ending-selector package

### ISSUE THREAD

**Issue by ellisfan** [SaschaNaz/AtomicChar#2]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.1.0
**System**: Unknown Windows Version
**Thrown From**: [AtomicChar](https://github.com/SaschaNaz/AtomicChar) package, v0.3.8
### Stack Trace

Failed to activate the line-ending-selector package

```
At this.getScrollWidth is not a function

TypeError: this.getScrollWidth is not a function
    at DisplayBuffer.atmcGetEditorWidth (C:\Users\Administrator\.atom\packages\AtomicChar\lib\atomic-wrapper.js:66:64)
    at DisplayBuffer.atmcGetSoftWrapWidth (C:\Users\Administrator\.atom\packages\AtomicChar\lib\atomic-wrapper.js:74:29)
    at DisplayBuffer.getSoftWrapColumn (C:\Users\Administrator\.atom\packages\AtomicChar\lib\atomic-wrapper.js:63:25)
    at DisplayBuffer.module.exports.DisplayBuffer.buildScreenLines (C:\Users\Administrator\AppData\Local\atom\app-1.1.0\resources\app.asar\src\display-buffer.js:1202:73)
    at DisplayBuffer.module.exports.DisplayBuffer.updateScreenLines (C:\Users\Administrator\AppData\Local\atom\app-1.1.0\resources\app.asar\src\display-buffer.js:1153:20)
    at DisplayBuffer.module.exports.DisplayBuffer.updateAllScreenLines (C:\Users\Administrator\AppData\Local\atom\app-1.1.0\resources\app.asar\src\display-buffer.js:238:19)
    at new DisplayBuffer (C:\Users\Administrator\AppData\Local\atom\app-1.1.0\resources\app.asar\src\display-buffer.js:114:12)
    at new TextEditor (C:\Users\Administrator\AppData\Local\atom\app-1.1.0\resources\app.asar\src\text-editor.js:89:30)
    at atom-text-editor.TextEditorElement.buildModel (C:\Users\Administrator\AppData\Local\atom\app-1.1.0\resources\app.asar\src\text-editor-element.js:168:28)
    at atom-text-editor.TextEditorElement.getModel (C:\Users\Administrator\AppData\Local\atom\app-1.1.0\resources\app.asar\src\text-editor-element.js:164:58)
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "autoHideMenuBar": true,
    "disabledPackages": [
      "csslint"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
AtomicChar, v0.3.8
autocomplete-paths, v1.0.2
autocomplete-php, v0.3.7
color-picker, v2.0.13
docblockr, v0.7.3
emmet, v2.3.13
file-icons, v1.6.11
jshint, v1.8.1
laravel, v0.7.0
minimap, v4.15.0
pretty-json, v0.4.1
wakatime, vundefined

# Dev
No dev packages
```

**Comment by CreatorMetaSky** [comment]
I solve this issue

``` sh
cd cd ~/.atom/packages
open .
rm -rf AtomicChar
```

then you can restart your atom, it's health, but the AtomicChar can't use.

ohh, you can also change the code of the wrapper.js at line 66, but i don't kown the right function name.

**Comment by saschanaz** [comment]
Just fixed by 404df2d1cbacc2b0694ba3eddd58fb9105b00e2f, thanks!

**Comment by CreatorMetaSky** [comment]
Perfect!
